tests/auth_tests/urls.py

```diff
@@ -198,7 +248,18 @@ def get_default_redirect_url(self):
         "login_and_permission_required_exception/",
         login_and_permission_required_exception,
     ),
+    path("public_view/", PublicView.as_view(), name="public_view"),
+    path("public_function_view/", public_view, name="public_function_view"),
+    path("public_async_view/", PublicAsyncView.as_view(), name="public_view"),
+    path("public_async_function_view/", public_async_view, name="public_function_view"),
+    path("protected_view/", ProtectedView.as_view(), name="protected_view"),
+    path("protected_function_view/", protected_view, name="protected_function_view"),
+    path("protected_async_view/", ProtectedAsyncView.as_view(), name="protected_view"),
+    path(
+        "protected_async_function_view/",
+        protected_async_view,
+        name="protected_function_view",
+    ),
     path("setlang/", set_language, name="set_language"),
-    # This line is only required to render the password reset with is_admin=True
```

Comment:
Please revert this line change.

Reply:
This statement is not true any more because we are testing admin redirect now. Should we remove that? Or can we go something along?
```
# This line is required to render the password reset with is_admin=True and for testing admin redirects under login required middleware
```

Reply:
Understood 👍